client/src/App.scss

```diff
@@ -12,3 +12,11 @@ html {
     margin: 0.5em 0.5em;
     text-align: center;
 }
+
+.tile.working {
+    background-color: #bffffb;
+}
+
+.tile.not_working {
+    background-color: #FBA39D;
```

Comment:
Maybe we can start defining colors as variables in another file for ease of use in another pr